src/amplitude-client.js

```diff
@@ -702,7 +702,7 @@ AmplitudeClient.prototype._saveReferrer = function _saveReferrer(referrer) {
 AmplitudeClient.prototype.saveEvents = function saveEvents() {
   try {
     if (AsyncStorage) {
-      AsyncStorage.setItem(this.options.unsentKey, JSON.stringify(this._unsentEvents));
+      AsyncStorage.setItem(this.options.unsentKey + this._storageSuffix, JSON.stringify(this._unsentEvents));
     } else {
       this._setInStorage(localStorage, this.options.unsentKey, JSON.stringify(this._unsentEvents));
```

Comment:
should append suffix here too as well as line 713, 715

Reply:
no need for line 715 bc `_setInStorage` is already appending `storageSuffix`

Reply:
oh oh 👍